src/Application/Regulation/Command/DuplicateRegulationCommandHandler.php

```diff
@@ -58,8 +58,11 @@ private function duplicateRegulationLocations(
     ): void {
         foreach ($originalRegulationOrder->getLocations() as $location) {
             $locationCommand = new SaveRegulationLocationCommand($duplicatedRegulationOrderRecord);
+            $locationCommand->roadType = $location->getRoadType();
             $locationCommand->cityCode = $location->getCityCode();
             $locationCommand->cityLabel = $location->getCityLabel();
+            $locationCommand->administrator = $location->getAdministrator();
+            $locationCommand->roadNumber = $location->getRoadNumber();
```

Comment:
(Détail) Dans la Command tu les déclares dans cet ordre (qui me semble préférable : les champs de chaque type sont "groupés" ensemble), alors mieux vaut le respecter ici ?

```suggestion
            $locationCommand->roadType = $location->getRoadType();
            $locationCommand->administrator = $location->getAdministrator();
            $locationCommand->roadNumber = $location->getRoadNumber();
            $locationCommand->cityCode = $location->getCityCode();
            $locationCommand->cityLabel = $location->getCityLabel();
```